jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java

```
@@ -200,7 +198,7 @@ public KogitoProcessInstance createProcessInstance(String processId,
 
     @Override
     public ProcessInstance startProcessInstance(long l) {
-        throw new UnsupportedOperationException();
+        return kogitoProcessRuntime.startProcessInstance(Long.toString(l));
```

the same, I'm not sure why do we need to change this now, because the idea to start a processInstance would be through the kogito-api like `kogitoProcessInstance.start()`, right?

---

in that case, I think you are right and is better to throw unsupported